app/scripts/modules/openstack/serverGroup/configure/wizard/capacity.html

```diff
@@ -0,0 +1,25 @@
+<form ng-controller="openstackServerGroupCapacityCtrl" class="container-fluid form-horizontal" name="form" novalidate>
```

Comment:
I think this file can be omitted - doesn't look like it's referenced anywhere and it looks like the older style of modal.

(Might be worth removing it from whichever provider this PR is based on (kubernetes, I think), too)